packages/snaps-sdk/src/internals/time.ts

```diff
@@ -0,0 +1,25 @@
+import { refine, string } from '@metamask/superstruct';
+import { DateTime } from 'luxon';
```

Comment:
Generally not a big fan of including this in the SDK, but I guess we don't have much of an option if we want to keep proper validation